platform/src/main/java/org/stellar/anchor/platform/event/KafkaSession.java

```diff
@@ -204,7 +206,22 @@ void configureAuth(Properties props) throws InvalidConfigException {
                 + SecretManager.getInstance()
                     .get(PropertySecretConfig.SECRET_EVENTS_QUEUE_KAFKA_PASSWORD)
                 + "\";");
+
+        break;
+    }
+
+    switch (kafkaConfig.getSecurityProtocol()) {
+      case SASL_PLAINTEXT:
+        props.put(SECURITY_PROTOCOL_CONFIG, "SASL_PLAINTEXT");
+        props.put(SASL_MECHANISM, kafkaConfig.getSaslMechanism().getValue());
         break;
+      case SASL_SSL:
+        props.put(SECURITY_PROTOCOL_CONFIG, "SASL_SSL");
+        props.put(SASL_MECHANISM, kafkaConfig.getSaslMechanism().getValue());
+        // Disable hostname verification
+        props.put("ssl.endpoint.identification.algorithm", "");
```

Comment:
Should we enable `HTTPS`?

Reply:
This only affects `Kafka`. I don't think we need `HTTPS`.

Reply:
Is there a reason not to enable this? This seems like an extra layer of security. Or at least make it a configurable property?

Reply:
Does `Kafka` use `HTTPS`?

Reply:
Yes, the default is `http` https://docs.confluent.io/platform/current/security/authentication/mutual-tls/overview.html#optional-settings

Reply:
Ok. Removed it to leave the default to `https`